monsteer/steering/plugin/nestSimulator.cpp

```diff
@@ -33,20 +34,21 @@ namespace steering
 
 namespace
 {
+const std::string nestSimulatorScheme = "nestSimulator";
```

Comment:
see my other reply: what is the right solution? In Brion the problem was different because the handles() methods of the plugin decided based on file extension. So should we place the string ids in a monsteer/config.h type of file?